ExomeDepth/README.md

```diff
@@ -1,14 +1,32 @@
 # Wrapper scripts for ExomeDepth (UMCU version)
 
 All script are made tested using Python 3.6.8
-## Make virtual python enviroment
-``` bash
-virtualenv -p python3 venv
-. venv/bin/activate
+## Installation 
+To run the ExomeDepth wrapper scripts we need to create a virtual python environment and folder with all the required R packages.
+
+### Make virtual python enviroment
+```bash
+python3 -m venv venv
+source venv/bin/activate
```

Comment:
pip install --upgrade pip
easy_install pip

Reply:
fixed in 6c8998d